File: src/main/java/io/junseok/mealmate/MealmateApplication.java
package io.junseok.mealmate;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class MealmateApplication {

	public static void main(String[] args) {
		SpringApplication.run(MealmateApplication.class, args);
	}

}

File: src/main/java/io/junseok/mealmate/MealmateApplication.kt
package io.junseok.mealmate

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class MealmateApplication
    fun main(args: Array<String>) {
        runApplication<MealmateApplication>(*args)
    }

